app/assets/javascripts/MathJax/mathjax_helpers.js

```diff
@@ -3,3 +3,25 @@
 function reloadDOM() {
     MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
 }
+
+function updateAnnotationPreview(){
+    var newContent = document.getElementById('new_annotation_content').value;
+
+    var preview_paragraph = document.getElementById('annotation_preview');
+    var title = document.getElementById('annotation_preview_title');
+
+    var first_index = newContent.indexOf("$$");
+    var second_index = newContent.indexOf("$$", first_index + 1);
+
+    if(first_index != -1 && second_index != -1 && first_index != second_index){
+        title.show();
+        preview_paragraph.show();
+        preview_paragraph.innerHTML = newContent;
+
+        // typeset the preview
+        MathJax.Hub.Queue(["Typeset",MathJax.Hub,preview_paragraph]);
```

Comment:
Identifier 'preview_paragraph' is not in camel case.<br>'MathJax' is not defined.